sys-auth/libfprint/files/libfprint-1.94.9-tod-fix-linking-with-lld.patch

```diff
@@ -0,0 +1,23 @@
+From 7619e0292b242a2628979b8317573c80a9f0bf6a Mon Sep 17 00:00:00 2001
+From: Brahmajit Das <listout@listout.xyz>
+Date: Sat, 20 Sep 2025 10:14:33 +0530
+Subject: [PATCH] fix fpi_byte_writer_change_pos not defined with clang
+
+On clang, linking with LLD fails.
+ld.lld: error: version script assignment of 'LIBFPRINT_TOD_1_1.94' to symbol 'fpi_byte_writer_change_pos' failed: symbol not defined
+clang: error: linker command failed with exit code 1 (use -v to see invocation)
+
+Signed-off-by: Brahmajit Das <listout@listout.xyz>
```

Comment:
Did you send this upstream? Please replace the description with a link to the upstream commit/merge request.

Reply:
I haven't, I'm not sure to apply this conditionally for LLD in a non-gentoo manner i.e. via the build system. Maybe I should at atleast open an issue upstream

Reply:
@Nowa-Ammerlaan https://gitlab.freedesktop.org/3v1n0/libfprint/-/issues/10

Reply:
Thanks, please replace the header in the patch with this link.

Reply:
Done